locale/pt-br/texts.po

```diff
@@ -970,6 +971,82 @@ msgstr "A wallet não está pronta para carregar endereços."
 msgid "There was an error while loading wallet addresses. Try again."
 msgstr "Ocorreu um erro durante download dos endereços da wallet. Tente novamente."
 
+#: src/screens/NanoContract/NanoContractRegisterScreen.js:50
+msgid "Nano Contract ID is required."
+msgstr "ID do Nano Contract é obrigatório."
+
+#: src/screens/NanoContract/NanoContractRegisterScreen.js:135
+msgid "See contract"
+msgstr "Ver contrato"
+
+#: src/screens/NanoContract/NanoContractRegisterScreen.js:153
+msgid "Load First Addresses Error"
+msgstr ""
+
+#: src/components/NanoContract/NanoContractDetails.component.js:158
+#: src/components/NanoContract/SelectAddressModal.component.js:105
+#: src/screens/NanoContract/NanoContractRegisterScreen.js:161
+msgid "Loading"
+msgstr "Carregando"
+
+#: src/screens/NanoContract/NanoContractRegisterScreen.js:162
+msgid "Loading first wallet address."
+msgstr "Carregando primeiro endereço da wallet."
+
+#: src/components/NanoContract/NanoContractDetailsHeader.component.js:142
+#: src/components/NanoContract/NanoContractTransactionHeader.component.js:90
+#: src/components/NanoContract/NanoContractsListItem.js:57
+#: src/components/TxDetailsModal.js:106
+#: src/screens/NanoContract/NanoContractRegisterScreen.js:171
+msgid "Nano Contract ID"
+msgstr "ID do Nano Contract"
+
+#: src/screens/NanoContract/NanoContractRegisterScreen.js:179
+msgid "Wallet Address"
+msgstr "Endereço da Carteira"
+
+#: src/screens/NanoContract/NanoContractRegisterScreen.js:189
+msgid "If you want to change the wallet address, you will be able to do"
+msgstr "Se deseja alterar o endereço de assinatura, você pode alterar"
+
+#: src/screens/NanoContract/NanoContractRegisterScreen.js:191
+msgid "after the contract is registered."
+msgstr "depois do contrato ser registrado."
+
+#: src/screens/NanoContract/NanoContractRegisterScreen.js:210
+msgid "Register Nano Contract"
+msgstr "Registrar Nano Contract"
+
+#: src/screens/NanoContract/NanoContractRegisterScreen.js:228
+msgid "Nano Contract Registration"
+msgstr "Registro do Nano Contract"
+
+#: src/screens/NanoContract/NanoContractTransactionScreen.js:37
+msgid "Nano Contract Transaction"
+msgstr "Transação do Nano Contract"
+
+#: src/screens/NanoContract/helper.js:11
+msgid "Contract successfully registered."
+msgstr "Nano Contract registrado com sucesso."
+
+#: src/sagas/mixins.js:33
```

Comment:
This file is gone now, is an update missing on the translations?

Reply:
Same for all below

Reply:
Removed.